docs/manual/advanced-features.tex

```diff
@@ -1233,6 +1233,8 @@
 \item \refqualclass{checker/nullness/qual}{RequiresNonNull}
 \item \refqualclass{checker/nullness/qual}{EnsuresNonNull}
 \item \refqualclass{checker/nullness/qual}{EnsuresNonNullIf}
+\item \refqualclass{checker/optional/qual}{EnsuresPresent}
```

Comment:
I suggest adding `RequiresPresent`, for completeness.  That can be in a separate pull request.